assets/js/translator.js

```diff
@@ -1289,15 +1289,23 @@ function setDefaultSrcLang() {
         return iso639Lang;
     }
 
+    function setLangs(lang) {
```

Comment:
This should already be done by the `restoreChoices('translator')`. This code shouldn't be messing with any of that logic.

Reply:
This function happens after `restoreChoices('translator')`, so i thought we should check again

Reply:
I think it wont mess with that as this function happens if `restoreChoices('translator')` doesnot happened.

Reply:
All this code happens before `restoreChoices('translator')`: https://github.com/apertium/apertium-html-tools/pull/352/files#diff-8331c23a07e5eb815c5b4393e81715efL467. This is really sketchy approach and will lead to hard to find bugs. Code outside that file shouldn't be touching local storage.